profile/README.md

```diff
@@ -0,0 +1,24 @@
+<h2 align="center">JSON Schema is a declarative language that allows you to annotate and validate JSON documents</h2>
+
+[![Slack](https://img.shields.io/static/v1?label=Slack&message=@json-schema&color=yellow)](https://asyncapi.slack.com)
+[![Twitter](https://img.shields.io/static/v1?label=Twitter&message=@jsonschema&color=9cf)](https://twitter.com/jsonschema)
+[![LinkedIn](https://img.shields.io/static/v1?label=LinkedIn&message=@jsonschema&color=lightgray)](https://www.linkedin.com/company/jsonschema)
+[![YouTube](https://img.shields.io/static/v1?label=Youtube&message=@JSONSchemaOrgOfficial&color=red)](https://www.youtube.com/@JSONSchemaOrgOfficial)
+[![Google Groups](https://img.shields.io/static/v1?label=Google&nbsp;Groups&message=@json-schema&color==blueviolet)](https://groups.google.com/g/json-schema)
+
+### 👋 Welcome
+
+Welcome to JSON Schema, a vocabulary that allows you to validate, annotate, and manipulate JSON documents.
```

Comment:
I'd consider reconciling this with the first sentence -- i.e. compare:

> JSON Schema is a declarative language that allows you to annotate and validate JSON documents

with

> [...] JSON Schema, a vocabulary that allows you to validate, annotate, and manipulate JSON documents.